pytradfri/device/__init__.py

```diff
@@ -136,48 +137,51 @@ class DeviceInfo:
     }
     ATTR_BATTERY = "9"
 
-    def __init__(self, device):
+    def __init__(self, device) -> None:
         """Create object of class."""
         self._device = device
 
     @property
-    def manufacturer(self):
+    def manufacturer(self) -> str:
         """Human readable manufacturer name."""
-        return self.raw.get(DeviceInfo.ATTR_MANUFACTURER)
+        return cast(str, self.raw.get(DeviceInfo.ATTR_MANUFACTURER))
```

Comment:
We can't cast to string and use dict.get at the same time. `dict.get` may return None.

Reply:
What is the best-practice way to go about here?
```
try:
    return cast(str, self.raw[DeviceInfo.ATTR_MANUFACTURER])
except KeyError:
    return None
```
or
```
if DeviceInfo.ATTR_MANUFACTURER in self.raw:
    return cast(str, self.raw[DeviceInfo.ATTR_MANUFACTURER])

return None
```

Or maybe another option that I'm unaware of?

Reply:
I think we should either use a TypedDict or pydantic to type the raw response so that we don't need to cast the type of the raw items in the properties here. I also believe that some of the device info raw items are not optional.

Reply:
Ok, I’ll look into that and see how it works.